README.md

```diff
@@ -316,3 +316,23 @@ The site will be accessible at http://localhost:1313.
 [terraform]: https://www.terraform.io/
 [AWS S3]: https://aws.amazon.com/s3/
 [Harvester manual test cases]: https://github.com/harvester/tests/tree/main/docs/content/manual
+
```

Comment:
Should we replace the Preview Website section above?

Reply:
Good point. I'll update that.

Reply:
Updated